.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
```

Comment:
I copied this from https://github.com/18F/identity-idp/blob/main/.github/dependabot.yml, but yanked the `identity-style-guide` bit since I wasn't sure if we wanted to update this as often